pages/_document.js

```diff
@@ -58,33 +59,25 @@ export default class Page extends Document {
               content="width=device-width, initial-scale=1.0, maximum-scale=1.0,
               user-scalable=0"
             />
-            <link
-              rel="shortcut icon"
-              type="image/x-icon"
-              href="/static/favicon.ico"
-            />
+            <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
             <link
               rel="shortcut icon"
               type="image/vnd.microsoft.icon"
-              href="/static/favicon.ico"
+              href="/favicon.ico"
             />
             <link
               rel="icon"
               type="image/png"
-              href="/static/favicon-32x32.png"
+              href="/favicon-32x32.png"
               sizes="32x32"
             />
             <link
               rel="icon"
               type="image/png"
-              href="/static/favicon-16x16.png"
```

Comment:
they all should be part of the `/public/static` ... may be introduce even `/public/static/img` or something similar

Reply:
probably the way favicons are set can be reviewed, but it looks it's good enough to be part of `static`, not `root`. May be it worth making them `static/img/favicon/...`

Reply:
Woops forgot to remove img/ and the favicons to static/

Reply:
OK, moved the loose image files (favicons, social-share.png) into img/, and will put /img/ back into the the/static/img/ path.